File: psra/service/src/main/java/by/arhor/psra/dto/TagDto.java
package by.arhor.psra.dto;

import java.io.Serializable;

import by.arhor.psra.CoreVersion;
import lombok.Data;

@Data
public class TagDto implements Serializable {
	
	private static final long serialVersionUID = CoreVersion.SERIAL_VERSION_UID;
	
	private String name;
	
}

File: psra/service/src/main/scala/by/arhor/psra/dto/TagDto.scala
package by.arhor.psra.dto

import java.io.Serializable

import by.arhor.psra.CoreVersion

import scala.beans.BeanProperty

object TagDto {
	val serialVersionUID: Long = CoreVersion.SERIAL_VERSION_UID
}

class TagDto extends Serializable {

	@BeanProperty var name: String = _
	
}
